### PR TITLE
Hooking up the juding score stepper

### PR DESCRIPTION
**Pull request by shaunxp20**
This will add very basic/rough functionality to get the judging screens working. There is still a bit of work to do (I added some more notes/todos to #3350), but this will at least give us the basic wiring up functionality and keep us moving forward.

### CODE REVIEW

File: app/views/judge/scores/new.en.html.erb [hunk 1]
```diff
@@ -11,8 +11,20 @@
...
+        <div class="w-full">
+        <% end %>
+
+        <router-view></router-view>
+
+        </div>
+      </div>
+    </div>
+  </template>
+</div>
```
Comment (viviancan) [inline app/views/judge/scores/new.en.html.erb:28]: I think there are a few extra `<div>`s here!

Comment (shaunxp20) [reply app/views/judge/scores/new.en.html.erb:28]: Yes! That is correct and intended! 😊😊😅  

This is what I had to do to make the layout display correctly! 😅😅😅

If I remove one/all of them the footer won't display correctly. I'm not sure why, we might have some `<div>` tags misaligned somewhere. I'll add this to #3350 for followup work to figure this out. 😬😁😅

**Review by devkonrad — APPROVED** [review]
I guess that the changes make sense and take care of great part of the  issue https://github.com/Iridescent-CM/technovation-app/issues/3350

File: app/views/judge/scores/new.en.html.erb (hunk 1 above)
Comment (viviancan) [reply app/views/judge/scores/new.en.html.erb:28]: This formatting issue was added to #3350 ! We will revisit this in the next step!

**Review by viviancan — APPROVED** [review]
Woo! thanks for merging all the work and taking care of the score stepper! Very excited to see everything come together! 🎉 

For anyone reading this in the future, we have a few follow up items to tackle which have been added to #3350

**Pull request merged**